publish/publish.proj

```diff
@@ -7,12 +7,19 @@
   <UsingTask TaskName="ListAzureBlobs" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
   <UsingTask TaskName="UploadToAzure" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
 
+  <PropertyGroup>
```

Comment:
Why isn't this using BuildToolsTaskDir property instead of directly doing this here?

Reply:
Where is that property defined? I'm not familiar with it

Reply:
It is in build.common.props from BuildTools.